src/shared/services/token-service.ts

```diff
@@ -0,0 +1,32 @@
+class TokenService {
+  private accessTokenKey: string;
+  private refreshTokenKey: string;
+
+  constructor() {
```

Comment:
constructor(accessTokenKey: string, refreshTokenKey: string)
this.accessTokenKey = accessTokenKey;
this.refreshTokenKey = refreshTokenKey;

Reply:
try not to hardcode any values inside classes, it's a bad practice

Reply:
resolved